### ISSUE TITLE
Missing "Unknown" Record properties are ignored

### ISSUE THREAD

**Issue by johngeorgewright** [pelotom/runtypes#182]
If you were to create a `Record` with 1 `Unknown` property, it seems to be ignored during validation:

```typescript
import { Record, Unknown } from 'runtypes'
const T = Record({ foo: Unknown })
console.info(T.check({}))
// {}
```

However, static typing would find this an error:

```typescript
import { Static } from 'runtypes'
const test: Static<typeof T> = {}
// Property 'foo' is missing in type '{}' but required in type '{ foo: unknown; }'. ts(2741)
```

**Comment by yuhr** [comment]
FWIW a related conversation is here: #113

**Comment by yuhr** [comment]
Resolved in #113, please check it in `v5.2.0`.
